components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/userstore/add-user-role.jsp

```diff
@@ -75,6 +75,7 @@
     </script>
     <%
     	UserRealmInfo userRealmInfo = null;
+        boolean multipleUserStores = false;
```

Comment:
Fix indentation. In JSPs at least they should be relatively correct. If you think others are wrong fix them.